cmake/Finddpdk.cmake

```diff
@@ -84,6 +84,14 @@ endif ()
 list (APPEND dpdk_REQUIRED
   dpdk_INCLUDE_DIR)
 
+# Depending on whether libbsd-dev exists at build time, DPDK may be built with a dependency on
+# libbsd. Note that other libraries in dpdk_PC_LIBRARIES are handled using separate logic
+# (see rte_libs above), thus the additional dependencies must be handled on a case by case basis.
```

Comment:
thanks for the comment. i admit that we should improve the code to either consume the `dpdk_PC_LIBRARIES` in a better way instead of hardwiring them.

Reply:
Thanks for mentioning this. I will check ways to improve this in a subsequent PR.